hardware_interface/include/hardware_interface/robot_hw.h

```diff
@@ -94,6 +94,18 @@ class RobotHW : public InterfaceManager
 
     return in_conflict;
   }
+/** \name Mode Switching
+   *\{*/
+
+  /**
+   * Check (in non-realtime) if given controller to be started and started can be switched from the current state of the RobotHW
+   */
+  virtual bool canSwitch(const std::list<ControllerInfo> &start_list, const std::list<ControllerInfo> &stop_list) const { return true; }
+
+  /**
+   * actually start and stop internal command handling (in non-realtime)
```

Comment:
Make the doc string more similar to that of `canSwitch`, so by reading both docstrings one can see that one method _checks_, and the other _does_. Mention that this method can't fail, that all checks should be implemented in `canSwitch`.